app/models/metrics/algorithms.rb

```diff
@@ -78,11 +79,25 @@ def count_matches(events)
     # @return does not return anything (the same matches Hash is passed in each loop but not explicitly sent back)
     def event_matches(event, matches)
       ids = match_standard_identifiers(event, matches)
+      citation = match_citations(event, matches)
       journal_exact = process_journals(event, matches)
       suggested_resource_exact = process_suggested_resources(event, matches)
       lcshs = match_lcsh(event, matches)
 
-      matches[:unmatched] += 1 if ids.detections.blank? && lcshs.detections.blank? && journal_exact.count.zero? && suggested_resource_exact.count.zero?
+      matches[:unmatched] += 1 if ids.detections.blank? && !citation.detection? && lcshs.detections.blank? && journal_exact.count.zero? && suggested_resource_exact.count.zero?
```

Comment:
I'd like to normalize `Citation` a bit more after seeing how it behaves differently. Not in this PR, but just eventually. This could take the form of adding the `.detection?` to all of our Detectors and/or normalizing what `.detection` returns. No changes are should be made as part of this work, but this difference made me pause and have to understand _why_ it was done this way. It makes sense based on how Citation currently works but feels like it may be confusing in the future as to why it is different.

Reply:
Yeah, I agree that having a more standardized set of methods across our detectors would be helpful. Some differences might be unavoidable (the citation detector doesn't extract specific patterns, while the standard identifier one does), but looking now I think the others (which all treat the phrase holistically) could benefit from a `.detection?` method.